java-manta-it/src/test/java/com/joyent/manta/client/multipart/EncryptedServerSideMultipartManagerIT.java

```diff
@@ -576,4 +578,27 @@ public final void canRetryUploadPart() throws IOException {
                     content, out.toByteArray());
         }
     }
+
+    public final void willFailToCreatePartWhenErrorOccursDuringUpload() throws Exception {
+        final String path = testPathPrefix + UUID.randomUUID().toString();
+        final byte[] content = RandomUtils.nextBytes(2048);
+
+        EncryptedMultipartUpload<ServerSideMultipartUpload> upload = multipart.initiateUpload(path);
+
+        Assert.assertThrows(IOException.class, () -> {
+            // partial read of content1
+            multipart.uploadPart(upload, 1, new FailingInputStream(new ByteArrayInputStream(content), 1024));
+        });
+
+        // listing the directory immediately after MPU creation is likely to return no results
+        // this is being reported upstream
+        TimeUnit.SECONDS.sleep(1);
```

Comment:
Shouldn't you do a loop around the sleep in case your sleep wait value is too small?

Reply:
This is a really weird context to try to explain..

Based on the fact that you need to see a test fail before it passes I was trying to design the test so it would fail without the proposed change (during which I temporarily reverted the fix). While running the test case I discovered that the uploaded part does not appear in the MPU part directory immediately after the PUT request completes, so the test case was passing both *with* and *without* the change. 

The [RFD](https://github.com/joyent/rfd/blob/master/rfd/0065/README.md#2-upload-part) seems to imply that there might be a race condition between upload-part and MPU state. I couldn't find anything that might imply directory listings would be affected though.

In summary, we don't really have anything to check in a loop since the expectation is that the part never appears. This test is starting to seem a bit contrived, I'll need to rethink whether it's worth keeping. Perhaps it would make more sense to create a unit test for `EncryptingPartEntity` like the test cases created for `EncryptingEntity`?